server/db/driver/pg/internal/matches.go

```diff
@@ -55,15 +55,14 @@ const (
 
 		-- participant/B (taker) REDEEM data
 		bRedeemCoinID BYTEA,
-		bRedeemTime INT8,         -- server time stamp
-		aSigAckOfBRedeem BYTEA   -- counterparty's (initiator) sig with ack of participant REDEEM data
+		bRedeemTime INT8          -- server time stamp
 	)`
 
 	RetrieveSwapData = `SELECT status, sigMatchAckMaker, sigMatchAckTaker,
 		aContractCoinID, aContract, aContractTime, bSigAckOfAContract,
 		bContractCoinID, bContract, bContractTime, aSigAckOfBContract,
 		aRedeemCoinID, aRedeemSecret, aRedeemTime, bSigAckOfARedeem,
-		bRedeemCoinID, bRedeemTime, aSigAckOfBRedeem
```

Comment:
Fortunately an explicit upgrade is not required since there are no `select *` queries where the number of columns is implicit.